common/include/pcl/common/time_trigger.h

```diff
@@ -43,6 +43,9 @@
 #include <boost/function.hpp>
 #include <boost/thread.hpp>
 #include <boost/signals2.hpp>
+
+#include <condition_variable>
```

Comment:
I'd expect that unlike Boost, these std headers don't cause trouble for Qt MOC. But this change is not worth re-running CI pipelines :D